plaso/parsers/locate.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""Parser for locate database (updatedb) files."""
+
+import os
+
+from dfdatetime import posix_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.lib import dtfabric_helper
+from plaso.lib import errors
+from plaso.lib import specification
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class LocateDatabaseEvent(events.EventData):
+  """Linux locate database (updatedb) event data.
+
+  Attributes:
+    folder_path (str): full folder path.
+  """
+
+  DATA_TYPE = 'linux:locate'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(LocateDatabaseEvent, self).__init__(data_type=self.DATA_TYPE)
+    self.folder_path = None
+
+
+class LocateDatabaseFile(dtfabric_helper.DtFabricHelper):
```

Comment:
I would merge LocateDatabaseFile and LocateDatabaseParser. In the Plaso code base there is no need for LocateDatabaseFile to be separate. We can change this in a follow up PR, and leave it as-is for now.

Reply:
Ok will do.  Thanks.